code/modules/events/crystal_event.dm

```diff
@@ -11,17 +11,16 @@ GLOBAL_LIST_INIT(crystal_invasion_waves, list(
 	"big wave" = list(
 		/obj/structure/crystal_portal/small=5,
 		/obj/structure/crystal_portal/medium=3,
-		/obj/structure/crystal_portal/big=2,
-		/obj/structure/crystal_portal/huge=1
+		/obj/structure/crystal_portal/big=2
 		),
 	"huge wave" = list(
 		/obj/structure/crystal_portal/small=7,
 		/obj/structure/crystal_portal/medium=5,
-		/obj/structure/crystal_portal/big=3,
-		/obj/structure/crystal_portal/huge=2
+		/obj/structure/crystal_portal/big=3
 		),
 	))
 GLOBAL_LIST_EMPTY(crystal_portals)
+GLOBAL_LIST_EMPTY(crystal_portals_center)
```

Comment:
Please document what these lists are about.
What's the difference between them?
Is the second list basically `huge_crystal_portals` ?

Reply:
i'll document them properly!

also yes the second is huge_crystal_portals, i guess i can just name it that way